docs/src/components/Layout/GlobalNav/GlobalNav.module.scss

```diff
@@ -22,11 +22,14 @@
 
   // Top level navbar element, create all of the variables that will be used throughout the unified navbar
   --navbar-border-color: var(--amplify-colors-neutral-80);
+  --navbar-padding-inline: var(--amplify-space-large);
   --navbar-current-property-color: #ff9900;
   --secondary-nav-hover-color: black;
   --secondary-nav-hover-background-color: var(--amplify-colors-neutral-90);
   --dev-center-nav-height: 3rem;
   --amplify-colors-font-hover: var(--amplify-colors-neutral-10);
+  --navbar-mobile-nav-padding: 12px;
+  --nav-links-spacing: 20px;
```

Comment:
shall we use `--amplify-space-<size>` ? Or at least, please don't use `px` here

Reply:
These are global nav docs specific variables, so they can't use all of our theme variables (not available on all other sites)

Reply:
If not available, let's just change the unit to `rem`?